src/test/java/CustomUserDAOTests.java

```diff
@@ -0,0 +1,48 @@
+import TTT.databaseUtils.CustomUserDAO;
+import TTT.users.CustomUser;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class CustomUserDAOTests {
+
+    CustomUserDAO testObject = new CustomUserDAO();
+    CustomUser testCustomUser = new CustomUser("test@mail.com","123");
+
+    @Test
+    public void saveCustomUserTest(){
+        Assertions.assertTrue(testObject.saveUser(testCustomUser));
+        testObject.deleteCustomUser("test@mail.com");
```

Comment:
Can you also test if the user was deleted?

Reply:
This is to make sure that no unnecessary objects are left in the database after this action. Because after testing objects do NOT disappear.